static/js/containers/VideoDetailPage.js

```diff
@@ -207,7 +213,7 @@ class VideoDetailPage extends React.Component {
 const mapStateToProps = (state, ownProps) => {
   const { videoKey } = ownProps;
   const { videos, commonUi, videoUi } = state;
-  const video = videos.data ? videos.data.get(videoKey) : null;
+  const video = videos.data && _.isFunction(videos.data.get) ? videos.data.get(videoKey) : null;
```

Comment:
I added this to avoid some error behavior. Seems like `videos.data` is normally a `Map`, but after deleting a video, `videos.data` is an empty object (`{}`). Not exactly sure why this happens. We might want to open up an issue to look into it